Xamarin.PropertyEditing.Mac/Controls/Custom/ProxyResponderButton.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AppKit;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class ProxyResponderButton : NSButton
+	{
+		public RowProxyResponder ProxyResponder { get; set; }
+
+		public override void KeyDown (NSEvent theEvent)
+		{
+			switch (theEvent.KeyCode) {
+			case (int)NSKey.Tab:
+				if (ProxyResponder != null) {
```

Comment:
In example, this whole implementation makes little sense if ProxyResponder is null. Pass it in as a constructor parameter and check it's not null at that point.

Reply:
Let's fix this small bits in the next PR changes improving the API.